### PR TITLE
(DOCSP-1109): Updated Tableau SSL Connections.

### PR DESCRIPTION
**Pull request by atsansone**
@steveren : Can you give this a quick pass? Bryan and I worked this out and I am just trying to integrate it. [Staged](https://docs-mongodbcom-staging.corp.mongodb.com/bi-connector/anthonysansone/DOCSP-1109/connect/tableau.html#connect-from-tableau-with-authentication-and-tls-ssl).

### CODE REVIEW

File: source/connect/tableau.txt [hunk 1]
```diff
@@ -27,33 +27,47 @@ Connect from Tableau with the C Authentication Plugin
 Connect from Tableau without Authentication or TLS/SSL
 ------------------------------------------------------
 
-Click on the :guilabel:`MongoDB BI Connector` named
-connection in Tableau's left hand
-column. Enter the host and port on which :program:`mongosqld` is
-listening, and click :guilabel:`Sign In` to connect.
+1. Click on the :guilabel:`MongoDB BI Connector` named connection in
+   Tableau's left hand column.
+2. Enter the host and port on which :program:`mongosqld` is listening
```
Comment (steveren) [inline source/connect/tableau.txt]: Suggest a period at the end for consistency's sake.

File: source/connect/tableau.txt [hunk 2]
```diff
@@ -27,33 +27,47 @@ Connect from Tableau with the C Authentication Plugin
...
+
+Using Tableau 10.3 or Later
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Tableau configures the connection for you.
+
+Using Tableau 10.2 or Earlier
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Tableau versions 10.2 or earlier, you must configure Tableau to send
```
Comment (steveren) [inline source/connect/tableau.txt]: First clause feels incomplete. Maybe "With Tableau versions 10.2..."

**Comment by breinero** [comment]
The image displayed within the connect from earlier versions of Tableau shows the user connecting with the "MongoDB BI Connector" named connection, rather than the "MySQL" named connection as mentioned in the text.

**Comment by atsansone** [comment]
@breinero : Can you take another pass and provide your changes or approval? [Staged](https://docs-mongodbcom-staging.corp.mongodb.com/bi-connector/anthonysansone/DOCSP-1109/connect/tableau.html#connect-from-tableau-with-authentication-and-tls-ssl)

**Comment by breinero** [comment]
The section Using Tableau 10.3 or Later is inaccurate. The only thing version 10.3 of Tableau does is set the Cleartext plugin, if the user is connecting with the named MongoDB connection. You still need to activate SSL by checking the checkbox and user the config GUI to select the right CA cert for TLS / SSL connections. Details are available here: https://docs.google.com/document/d/1Tc9ALQsqcIRxUa6sKSY3cdqfQZn_44_uspY698rKS-c/edit#heading=h.cnsstjh0jn9g

**Comment by breinero** [comment]
LGTM thanks Tony and Steve

**Comment by atsansone** [comment]
cb50db3e0b4ebf8ddce413eb6c58bc2e1c6b71b6

**Pull request closed without merging**